idea_town/frontend/static-src/app/models/experiment.js

```diff
@@ -1,9 +1,28 @@
+import app from 'ampersand-app';
 import Model from 'ampersand-model';
 
 export default Model.extend({
   urlRoot: '/api/experiments',
   extraProperties: 'allow',
-
+  props: {
+    enabled: {type: 'boolean', default: false}
+  },
   // This shouldn't be necessary; see comments in collections/experiments.js
-  ajaxConfig: { headers: { 'Accept': 'application/json' }}
+  ajaxConfig: { headers: { 'Accept': 'application/json' }},
+
+  // TODO(DJ): this will be removed when we start tracking
+  // install state through the User Installation model.
+  initialize() {
+    app.on('addon-install:install-ended', addon => {
+      if (addon.id === this.id) {
+        this.enabled = true;
```

Comment:
We currently have the addon setup so we uninstall on disable. Should we keep them installed?